bgp-monitor-action-pack/bgp-monitor/script.py

```diff
@@ -55,6 +56,31 @@ def IsStatsDiffExpected(prevStats, currentStats, expectedDiff: int):
         err = "Missing change control device" if device is None \
             else f"device {device} is missing 'id'"
         raise ActionFailed(err)
+
+    # This block checks for the check_established flag, defaults to False
+    # If set to True, this block will check that all BGP peers are "Established"
+    # If any BGP peers are not Established, the CC will abort/fail
+    if checkEstablished:
+        bgpPeerState = ctx.runDeviceCmds(['enable', 'show ip bgp summary vrf all', 'show bgp evpn summary'])
+        errs = [resp.get('error') for resp in bgpPeerState if resp.get('error')]
+        if errs:
+            raise ActionFailed(f"Running action failed with: {errs[0]}")
+
+        for bgppeer in bgpPeerState[1]["response"]["vrfs"]["default"]["peers"]:
```

Comment:
Do we only ever care about the default vrf? Should "vrf" be a flag to the action? Or should there be a "checkAllVrfs" flag? If false you just check default.

Reply:
The user has specifically asked to look in all VRFs and EVPN AF. Perhaps I need to re-work this loop if I am not actually looking outside the default VRF in how I have written the for loop with regard to bgpPeerState[1]["response"]**["vrfs"]["default"]**["peers"]. I will talk with you more about this on chat.

Reply:
for now I will check all VRFs. I have made changes to the loop so it encompasses all vrfs and all peers in each vrf. This update will be in my next commit.